libraries/state_db/detail/state_delta.cpp

```diff
@@ -2,6 +2,8 @@
 
 #include <koinos/crypto/merkle_tree.hpp>
 
+#include <koinos/protocol/protocol.pb.h>
```

Comment:
If `state_delta.hpp` defines `get_delta_entries` successfully, I don't think including the header in the source file is needed.